packages/react-examples/src/charting/GroupedVerticalBarChart/GroupedVerticalBarChart.Basic.Example.tsx

```diff
@@ -7,7 +7,7 @@ export class GroupedVerticalBarChartBasicExample extends React.Component<Readonl
   public render(): React.ReactNode {
     const data = [
       {
-        name: 'LongLong text here It should Display all',
+        name: `Data 1`,
```

Comment:
It looks like the purpose of this item was to show that something works for long text, so you should still use some other long text here.